jans-cedarling/bindings/cedarling_c/tests/test_cedarling.c

```
@@ -667,28 +667,237 @@ void test_trusted_issuer_loading_info() {
+    TEST_ASSERT(ret != 0, "Empty items batch is rejected");
+    TEST_ASSERT(auth_result.error_message != NULL, "Empty items error has message");
+    if (auth_result.error_message != NULL) {
+        // Rust-side error string includes "Empty items array".
+        TEST_ASSERT(strstr(auth_result.error_message, "mpty") != NULL,
+                    "Error mentions empty items");
+    }
+    cedarling_free_result(&auth_result);
+
+    // NULL parameter guards.
+    ret = cedarling_authorize_unsigned_batch(instance_id, NULL, &auth_result);
+    TEST_ASSERT(ret != 0, "Reject NULL batch JSON");
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)